packages/ui/src/stories/Button/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+/* import './button.css'; */
```

Comment:
I think you might not need this

Reply:
I deleted. My bad I missed that. I already practiced configuring prettier with eslint-config-airbnb-typescript so that will help me not those silly mistakes. I am also thinking about adding husky to save you work and make me be more clean and precise with my commits. Do you think that is a good idea? I have never used but I found it doing research on the eslint/prettier issue.